sonar-db/src/test/java/org/sonar/db/user/UserDaoTest.java

```diff
@@ -469,65 +486,71 @@ public void exists_by_email() throws Exception {
   }
 
-  private UserDto newActiveUser(){
+  private UserDto newActiveUser() {
     return newUser(true);
   }
 
-  private UserDto newUser(boolean active){
+  private UserDto newUser(boolean active) {
     UserDto dto = newUserDto().setActive(active);
     underTest.insert(session, dto);
     return dto;
   }
 
-  private DashboardDto newDashboard(UserDto user, boolean shared){
+  private DashboardDto newDashboard(UserDto user, boolean shared) {
     DashboardDto dto = new DashboardDto().setUserId(user.getId()).setShared(shared);
     dbClient.dashboardDao().insert(session, dto);
     return dto;
   }
 
-  private ActiveDashboardDto newActiveDashboard(DashboardDto dashboard, UserDto user){
+  private ActiveDashboardDto newActiveDashboard(DashboardDto dashboard, UserDto user) {
     ActiveDashboardDto dto = new ActiveDashboardDto().setDashboardId(dashboard.getId()).setUserId(user.getId());
     dbClient.activeDashboardDao().insert(session, dto);
     return dto;
   }
 
-  private IssueFilterDto newIssueFilter(UserDto user, boolean shared){
+  private IssueFilterDto newIssueFilter(UserDto user, boolean shared) {
     IssueFilterDto dto = new IssueFilterDto().setUserLogin(user.getLogin()).setName(randomAlphanumeric(100)).setShared(shared);
     dbClient.issueFilterDao().insert(session, dto);
     return dto;
   }
 
-  private IssueFilterFavouriteDto newIssueFilterFavourite(IssueFilterDto filter, UserDto user){
+  private IssueFilterFavouriteDto newIssueFilterFavourite(IssueFilterDto filter, UserDto user) {
     IssueFilterFavouriteDto dto = new IssueFilterFavouriteDto().setUserLogin(user.getLogin()).setIssueFilterId(filter.getId());
     dbClient.issueFilterFavouriteDao().insert(session, dto);
     return dto;
   }
 
-  private MeasureFilterDto newMeasureFilter(UserDto user, boolean shared){
+  private MeasureFilterDto newMeasureFilter(UserDto user, boolean shared) {
     MeasureFilterDto dto = new MeasureFilterDto().setUserId(user.getId()).setName(randomAlphanumeric(100)).setShared(shared);
     dbClient.measureFilterDao().insert(session, dto);
     return dto;
   }
 
-  private MeasureFilterFavouriteDto newMeasureFilterFavourite(MeasureFilterDto measureFilter, UserDto user){
+  private MeasureFilterFavouriteDto newMeasureFilterFavourite(MeasureFilterDto measureFilter, UserDto user) {
     MeasureFilterFavouriteDto dto = new MeasureFilterFavouriteDto().setUserId(user.getId()).setMeasureFilterId(measureFilter.getId());
     dbClient.measureFilterFavouriteDao().insert(session, dto);
     return dto;
   }
 
-  private PropertyDto newProperty(UserDto user){
+  private PropertyDto newProperty(UserDto user) {
     PropertyDto dto = new PropertyDto().setKey(randomAlphanumeric(100)).setUserId(user.getId());
     dbClient.propertiesDao().insertProperty(session, dto);
     return dto;
   }
 
-  private UserRoleDto newUserRole(UserDto user){
+  private PropertyDto newProperty(String key, String value, long componentId) {
```

Comment:
- name this method `insertProperty` instead of `newProperty`. new is for object creation, insert for insertions in datastore. I know the other methods are named like that
- commit in the method, so we don't have to think about it in the test method